groot/rsrv/rsrv_test.go

```diff
@@ -25,17 +24,13 @@ import (
 	"time"
 
 	uuid "github.com/hashicorp/go-uuid"
-	"go-hep.org/x/hep/hplot/cmpimg"
+	"gonum.org/v1/plot/cmpimg"
```

Comment:
@sbinet, What advanctage was `"gonum.org/v1/plot/cmpimg` providing that `go-hep.org/x/hep/hplot/cmpimg` was not providing? Just a curious question :)